lib/bootloader_pvm.pm

```diff
@@ -118,11 +118,10 @@ sub prepare_pvm_installation {
     # Delete partition table before starting installation
     select_console('install-shell');
 
-
     my $disks = script_output('lsblk -n -p -l -o NAME -d -e 7,11');
```

Comment:
can you rebase here to get the changes from https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9611/files?